app/components/primary_dashboard_programme_community_activity_group_component.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class PrimaryDashboardProgrammeCommunityActivityGroupComponent < CmsWithAsidesComponent
+  delegate :current_user, to: :helpers
+
+  def initialize(title:, programme_activity_group:, current_user:, aside_slug: nil)
+    aside_sections = if aside_slug.nil?
+      nil
+    else
+      [{slug: aside_slug}]
+    end
+
+    super(aside_sections:)
+    @title = title
+    @programme_activity_group = programme_activity_group
+  end
+
+  def before_render
+    @current_user = current_user
+  end
+
+  def user_programme_activities
```

Comment:
Would it be worth just having these as separate variables, rather than passing as a hash. Would mean refactoring the component its going into, but might improve readability on the component?